site.yml

```diff
@@ -76,6 +81,19 @@
       copy: src=jobs dest=/var/lib/jenkins/jobs/default-dsl-job/workspace directory_mode=0750 mode=0640 owner=jenkins group=jenkins
     - name: generate jenkins job from dsl seed job
       shell: "curl -X POST 'http://127.0.0.1:8080/job/default-dsl-job/build' -k $(cat ~/jenkins_auth)"
+    - name: create jenkins .ssh directory
+      file: path=/var/lib/jenkins/.ssh state=directory mode=0775 owner="jenkins"
+    - name: deploy deployer keys
+      copy: content="{{ deployer_key['key'] }}" dest=/var/lib/jenkins/.ssh/{{ deployer_key['name'] }} mode=0600 owner="jenkins"
```

Comment:
This line specifically I don't want to manipulate, as I'd have to use different form of specifying copy module parameters, which btw. has different rendering and I'd have to test all again if it still works.